services/print/docs/Print.md

```diff
@@ -0,0 +1,18 @@
+Print
+=======
+
+POST /print/
+-----------------
+
+Post a new print job to the SNS queue
+
+Request Format:
+
+```
+{
+	"id": "343434",
+	"email": "test@hackillinois.org"
+}
+```
+
+Can throw a PrintError
```

Comment:
I meant that the `Type` field in the error once its added should have `PrintError` which lets consumers have easy error checking instead of parsing the error message in generic errors.

Reply:
Oh okay. Is that part of the improved error handling mechanism you're developing?